### ISSUE TITLE
Better control of modal page layout

### ISSUE THREAD

**Issue by j4zzcat** [newrelic/nr1-status-widgets#26]
It would be nice to have more control over the layout of charts in the modal page.
As a minimum, I'd like to be able to have a singe table chart maxed-out on the entire modal page.

**Comment by Kav91** [comment]
Yes that is a good idea, I have 2 ideas I'm thinking about. Let me know your thoughts/suggestions as well.

Option 1:
- Allow the user to define the number of columns

Option 2:
- Use some automatic logic, perhaps:
1 query - max modal
2 queries - split half/half
3 queries and above - tile in columns of 3

**Comment by j4zzcat** [comment]
Option 1 seems to provide more control.
I don't know if that's possible at all, but maybe there's a way to load another dashboard's page as the modal?

**Comment by j4zzcat** [comment]
Sounds good.

**Comment by Kav91** [comment]
@j4zzcat you will see the new functionality once the below PR is merged to update the catalog version
https://github.com/newrelic/nr1-catalog/pull/472
